include/aws/auth/credentials.h

```diff
@@ -1009,6 +1024,20 @@ struct aws_credentials_provider *aws_credentials_provider_new_ecs(
     struct aws_allocator *allocator,
     const struct aws_credentials_provider_ecs_options *options);
 
+/**
+ * Creates a provider that sources credentials from the ecs role credentials service and reads the required params from
+ * environment variables
```

Comment:
tweak comments on aws_credentials_provider_new_ecs() to mention the alternative environment function. yes, this is also mentioned in the options-struct, but it doesn't hurt to be repetitive with important information like that

Reply:
Thanks, updated.